test/external/imports/ts-cjs/tsconfig.json

```diff
@@ -4,12 +4,11 @@
 	"compilerOptions": {
 		"target": "esnext",
 		"module": "commonjs",
-		"declaration": true,
+		"declaration": false,
 		"sourceMap": false,
-		"outDir": "build/",
 		"strict": true,
-		"moduleResolution": "node",
 		"esModuleInterop": true,
```

Comment:
what happens if we turn off `esModuleInterop`?

Reply:
We get the following when running the `ts-cjs` test:

```
    test/server-test.ts:4:8 - error TS1259: Module '"/data/code/contrib/express-rate-limit/core/test/external/imports/ts-cjs/node_modules/@types/supertest/index"' can only be default-imported using the 'esModuleInterop' flag

    4 import request from 'supertest'
             ~~~~~~~

      node_modules/@types/supertest/index.d.ts:10:1
        10 export = supertest;
           ~~~~~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
    test/server-test.ts:4:8 - error TS1259: Module '"/data/code/contrib/express-rate-limit/core/test/external/imports/ts-cjs/node_modules/@types/supertest/index"' can only be default-imported using the 'esModuleInterop' flag

    4 import request from 'supertest'
             ~~~~~~~

      node_modules/@types/supertest/index.d.ts:10:1
        10 export = supertest;
           ~~~~~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

We could use `const request = require('supertest')` instead in those files... that seems to fix the problem

Reply:
Fixed - https://github.com/nfriedly/express-rate-limit/pull/282#issuecomment-1006397135 🎉

Reply:
Nope not fixed 😕

Reply:
We need this so that we can do 

```ts
import createServer from 'express'

const app = createServer()
```

A possible fix is to use `import createServer = require('express')`

Reply:
Even `import createServer = require('express')` doesn't work :/